cdts/chkconfig-cos6-x86_64/meta.yaml

```diff
@@ -0,0 +1,38 @@
+package:
+  name: chkconfig-cos6-x86_64
+  version: 1.3.49.5
+
+source:
+  - url: http://mirror.centos.org/centos/6.10/os/x86_64/Packages/chkconfig-1.3.49.5-1.el6.x86_64.rpm
+    sha256: 78386af88e956364b220b3ef9fb268c7cf83ec1e8b763e65ae1cf71d4d4cc9dd
+    no_hoist: true
+    folder: binary
+  # - url: http://vault.centos.org/6.10/os/Source/SPackages/chkconfig-1.3.49.5-1.el6.src.rpm
+  #   no_hoist: true
+  #   folder: source
+
+build:
+  number: {{ cdt_build_number|int + 1000 }}
+  noarch: generic
+  missing_dso_whitelist:
+    - '*'
+
+requirements:
+  run:
+    - sysroot_linux-64 2.12
+
+about:
+  home: 
+  license: GPLv2
+  license_family: GPL2
+  # license_file: in the rpm source
```

Comment:
One concern that has come up before is the need to package the license file in the case of CDTs. While in theory this should be packaged somewhere (like the source RPM), but that doesn't always seem to be the case ( https://github.com/numactl/numactl/issues/68 ).

https://github.com/conda-forge/conda-forge.github.io/issues/783#issuecomment-506861301

Reply:
I don't see any refs to source RPMs in the comments above. We are copying over the entire RPM and for the ones I have checked, the license file is packaged there.

Reply:
Note that the current CDT skeleton code in conda build downloads the source RPMs if they exist but then doesn't actually copy them anywhere in the PREFIX. Thus we are not actually using them for licenses. The binary RPMs actually have the licenses in them.

Reply:
We could add code to look for license files and error unless we tell it to ignore the error.

Reply:
OK @jakirkham. The code now packages the license file or errors if it cannot find one.